source/_changelogs/2017-10-01-October.md

```diff
@@ -0,0 +1,29 @@
+---
+title: October 2017
+changelog: true
+---
+
+## Platform Improvements
+
+### Custom Upstreams
+We’ve opened access to Custom Upstreams for all organizations on Pantheon. Read more about it on our [blog](https://pantheon.io/blog/announcing-new-pantheon-upstream-workflow){.external}, or [give it a try](/docs/custom-upstream/).
+
+### Terminus 1.6
+If you haven’t updated Terminus recently, check out the <a href="/docs/terminus/updates#changelog" data-proofer-ignore>Terminus Changelog</a> to see what’s new and update to the <a href="/docs/terminus/updates#update-to-the-current-release-" data-proofer-ignore>current release</a>.
+
+### WebP Image Support
+[WebP](https://developers.google.com/speed/webp/){.external}, a new image format created by Google, is now supported by Pantheon’s Global CDN.
```

Comment:
What does supported mean in this context? How about replacing supported with cached?